test/spec/call_indirect_refnull.wast

```diff
@@ -0,0 +1,12 @@
+(module
+  (table $t 1 1 funcref)
+  (elem (table $t) (i32.const 0) funcref (ref.null func))
+
+  (func $call-refnull (export "call-refnull") (result f32)
+    (call_indirect (result f32) (i32.const 0))
+  )
+)
+(assert_trap
+  (invoke "call-refnull")
+  "uninitialized table element"
+)
```

Comment:
@kripken @tlively I added this to test `call_indirect` on `ref.null`. Is this a good way to do it, or should I go with another method?

Reply:
this test looks good to me.